dcos_launch/__init__.py

```diff
@@ -16,4 +17,9 @@ def get_launcher(config):
             return dcos_launch.onprem.OnpremLauncher(config)
     if platform == 'azure':
         return dcos_launch.arm.AzureResourceGroupLauncher(config)
+    if platform == 'gce':
+        if provider == 'onprem':
+            return dcos_launch.onprem.OnpremLauncher(config)
+        else:
+            raise dcos_launch.util.LauncherError('UnsupportedAction', 'Launch provider {} not supported for the {} platform.'.format(provider, platform))
```

Comment:
Yes, that's what I meant: no else, just the error with both provider and platform.